ajax/libs/PKI.js/package.json

```diff
@@ -5,7 +5,7 @@
   "version": "1.2.1",
   "repository": {
     "type": "git",
-    "url": "git://github.com/GlobalSign/PKI.js.git"
+    "url": "git://github.com/PeculiarVentures/PKI.js.git"
   },
   "homepage": "http://pkijs.org",
```

Comment:
How about also update the homepage?
`https://pkijs.org/`, using https is better because it's more security for browsing. To prevent the possibility the users been attacked during the redirection.